scripts/fast-reboot

```diff
@@ -533,9 +528,11 @@ sonic_asic_type=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
 BOOT_TYPE_ARG="cold"
 case "$REBOOT_TYPE" in
     "fast-reboot")
+        check_warm_restart_in_progress
         BOOT_TYPE_ARG=$REBOOT_TYPE
-        trap clear_fast_boot EXIT HUP INT QUIT TERM KILL ABRT ALRM
+        trap clear_boot EXIT HUP INT QUIT TERM KILL ABRT ALRM
         sonic-db-cli STATE_DB SET "FAST_REBOOT|system" "1" "EX" "180" &>/dev/null
+        config warm_restart enable system
```

Comment:
For people who does not know the background, it is really confuse here, I would suggest to add comment here to explain why fastboot requires "enable warmboot".

Reply:
Agree, it confused me.